decidim-proposals/lib/decidim/proposals/engine.rb

```diff
@@ -87,6 +87,12 @@ class Engine < ::Rails::Engine
           )
         end
       end
+
+      initializer "decidim_proposals.add_cells_view_paths" do
+        Cell::ViewModel.view_paths << File.expand_path("#{Decidim::Proposals::Engine.root}/app/cells")
+        # Cell::ViewModel.view_paths << File.expand_path("#{Decidim::Core::Engine.root}/app/cells") # for shared partials
```

Comment:
Check this commented line (remove it if it's not needed)

Reply:
removed, It's not needed since `Decidim::Proposals::ViewModel < Decidim::ViewModel`

it's in `core` https://github.com/decidim/decidim/blob/795eb8173c70f2651bbf9d05c59863bb0a480961/decidim-core/lib/decidim/core/engine.rb#L216